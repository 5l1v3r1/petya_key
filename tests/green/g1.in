a5PhwcoxzctW7vUN5B5oPLTf64cnTe4ZWJyx5tKGHhHJQvSFMfPNAHiZcazmSdz6ZWo4uaYKtmC3rQj6g67wCmrZDu918bA1
