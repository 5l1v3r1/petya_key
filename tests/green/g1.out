H9UM88r4k49QWmNk
